velox/exec/tests/QueryAssertionsTest.cpp

```diff
@@ -493,4 +494,23 @@ TEST_F(QueryAssertionsTest, intervalDayTime) {
   assertQuery(plan, "SELECT * FROM tmp");
 }
 
+TEST_F(QueryAssertionsTest, equalPlans) {
+  VectorFuzzer::Options opts;
+  opts.containerVariableLength = false;
```

Comment:
Is this necessary? I assume not.

Reply:
It was needed for the Unnest plans. But not for others. Have removed it now.